postgres/init.sls

```diff
@@ -252,7 +272,7 @@ postgresql-ext-{{ ext_name }}-for-db-{{ name }}:
       - postgres_user: postgresql-user-{{ ext.user }}
     {% endif %}
     {% if ext.get('schema') %}
-      - postgres_schema: postgresql-schema-{{ ext.schema }}
+      - postgres_schema: postgresql-schema-{{ ext.schema }}-for-db-{{ name }}
```

Comment:
This used to reference the same state at https://github.com/saltstack-formulas/postgres-formula/blob/master/postgres/init.sls#L192 , however the state id was simply wrong and resulted in a failing state call.